bambi/plots/utils.py

```diff
@@ -12,25 +14,74 @@
 
 
 @dataclass
-class ContrastInfo:
+class VariableInfo:
```

Comment:
Can you document this dataclass? I'm not so sure that thorough documentation of all the arguments is needed, but it would be nice to know what's the goal of this dataclass and where it is used.